testlibs.versions.toml

```diff
@@ -1,10 +1,10 @@
 [versions]
-mockbukkit = "3.9.0"
-slf4j = "2.0.7"
-junit = "5.10.0"
+mockbukkit = "update-1.21-SNAPSHOT"
+slf4j = "2.0.13"
+junit = "5.11.0-M2"
```

Comment:
im not really sure if we should update to unstable builds of junit if the old ones work just fine